### PR TITLE
Added the tasks to retrieve cert project and vendor related data

### PR DESCRIPTION
**Pull request by MarcinGinszt**
Related PR: https://github.com/redhat-openshift-ecosystem/operator-pipelines-images/pull/20
Jira task: ISV-989

### CODE REVIEW

File: ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ kind: Pipeline
 metadata:
   name: operator-hosted-pipeline
 spec:
-  # TODO: Clean-up git-related params when we'll start using webhooks. Many of them are redundant.
```
Comment (MarcinGinszt) [inline ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml:7]: I've created a Jira task for that: https://issues.redhat.com/browse/ISV-1054

File: ansible/roles/operator-pipeline/templates/openshift/tasks/cert-project-check.yml [hunk 2]
```diff
@@ -7,22 +7,11 @@ spec:
   params:
     - name: bundle_path
       description: path indicating the location of the certified bundle within the repository
-    - name: pyxis_cert_path
```
Comment (MarcinGinszt) [inline ansible/roles/operator-pipeline/templates/openshift/tasks/cert-project-check.yml:10]: The step cert-project-check is used by all pipelines to retrieve the cert_project_id from ci.yaml.
Lately, to this script there was added functionality to query Pyxis and retrieve cert project by id.
It is not needed in eg. CI pipeline. Also, it demands ssh-credentials, which should not be needed in the CI pipeline. I had to split this task.

File: ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml [hunk 3]
```diff
@@ -92,33 +93,57 @@ spec:
           subPath: src
 
     # Get additional data necessary for certification
```
Comment (haripate) [inline ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml:95]: Do we still need this comment?

File: ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+          value: $(params.pyxis_key_path)
+      script: |
+        #! /usr/bin/env bash
+        set -xe
+
+        get-cert-project-related-data \
+          --pyxis-url $(params.pyxis_url) \
+          --cert-project-id $(params.cert_project_id)
+
+        cat isv_pid | tee $(results.isv_pid.path)
```
Comment (Allda) [inline ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml:45]: How about store a project data as JSON that comes from Pyxis and use a `jq` to return a specific field. In the current way, each field needs to be stored in a separate file which is hard to maintain or extend in the future.

File: ansible/roles/operator-pipeline/templates/openshift/tasks/get-vendor-related-data.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+          value: $(params.pyxis_key_path)
+      script: |
+        #! /usr/bin/env bash
+        set -xe
+
+        get-vendor-related-data \
+          --pyxis-url $(params.pyxis_url) \
+          --org-id $(params.org_id)
+
+        cat vendor_label | tee $(results.vendor_label.path)
```
Comment (Allda) [inline ansible/roles/operator-pipeline/templates/openshift/tasks/get-vendor-related-data.yml:38]: The same here.

**Review by haripate — APPROVED** [review]
LGTM

**Review by Allda — APPROVED** [review]

**Pull request merged**